presentation/src/main/java/com/cointrend/presentation/ui/about/AboutScreen.kt

```diff
@@ -0,0 +1,236 @@
+package com.cointrend.presentation.ui.coinslist
+
+import android.content.Intent
+import android.net.Uri
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.gestures.detectTapGestures
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.modifier.modifierLocalConsumer
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cointrend.presentation.R
+import com.cointrend.presentation.commoncomposables.*
+import com.cointrend.presentation.theme.StocksDarkBackgroundTranslucent
+import com.cointrend.presentation.theme.StocksDarkSecondaryText
+import com.cointrend.presentation.ui.coindetail.SectionInfoItem
+
+
+private val defaultHorizontalPadding = 16.dp
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AboutScreen(
+) {
+    val focusManager = LocalFocusManager.current
+
+    Scaffold(
+        modifier = Modifier,
+        topBar = {
+            TopAppBar(
+                title = {IconButton(onClick = {
+                    System.out.println("Go Back")
+                }) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_arrow_back_ios),
+                        contentDescription = "Return to previous screen",
+                        modifier = Modifier.padding(start = 12.dp)
+                    )
+                }
+                }
+            )
+        }
+    ) { padding ->
+        Column(
```

Comment:
This outer Column seems to me useless. Try to remove it and move the attributes to the LazyColumn below